.eslintrc.cjs

```diff
@@ -187,6 +188,9 @@ module.exports = {
     "regexp/no-empty-alternative": "off",
     "regexp/no-super-linear-backtracking": "off",
 
+    "simple-import-sort/imports": isCI ? "error" : "warn",
+    "simple-import-sort/exports": isCI ? "error" : "warn",
```

Comment:
I prefer using `"error"` in even non CI environment.

Reply:
I agree with @sosukesuzuki , since this rule autofixable, "error" should be used.

Reply:
Done in https://github.com/prettier/prettier/pull/15817/commits/49394466ed48ee3971c264322de483ed30f785fe

Note that we still have `isCI` here:

https://github.com/prettier/prettier/blob/863368e1ba915eb885eca20c493ddad740cb8df2/.eslintrc.cjs#L35

I used to use `"error"` everywhere, but my colleague persuaded me to switch to `"warn"` for rules like these. If something may lead to a crash, it’s marked as an error. If something is just not good enough for a merge, it’s a warning.

Both warnings and errors return non-zero exit code. Here are our `package.json` scripts:

```
    "fix:eslint": "eslint --max-warnings=0 --fix \"**/*\"",
    "...": "...",
    "lint:eslint": "eslint --max-warnings=0 \"**/*\"",
```

When it comes to `"simple-import-sort/imports"` using `"warn"` is quite handy. It allows us to see incorrect order as well as broken imports within. I was at first hesitant to use `"warn"` at all because I did not know about `--max-warnings=0`. But these two levels of feedback make total sense to me now.

Reply:
I think current settings for `no-console` is fine too.

Story behind it https://github.com/prettier/prettier/pull/10322#issuecomment-779833246